node-template/runtime/build.rs

```diff
@@ -19,7 +19,10 @@ use wasm_builder_runner::{build_current_project_with_rustflags, WasmBuilderSourc
 fn main() {
 	build_current_project_with_rustflags(
 		"wasm_binary.rs",
-		WasmBuilderSource::Crates("1.0.4"),
+		WasmBuilderSource::CratesOrPath {
+			path: "../../core/utils/wasm-builder",
+			version: "1.0.4",
```

Comment:
@bkchr please check this

Reply:
This shall be reverted (I am patching wasm with it, but when trie will be commit this won't be necessary anymore, I should have put a comment).